packages/expo-secure-store/android/src/main/java/expo/modules/securestore/SecureStoreModule.java

```diff
@@ -128,7 +128,7 @@ private void setItemImpl(String key, String value, ReadableArguments options, Pr
     }
 
     String encryptedItemString = encryptedItem.toString();
-    if (encryptedItemString == null) {
+    if ("".equals(encryptedItemString)) {
```

Comment:
> Make a JSON text of this JSONObject. For compactness, no whitespace is added. If this would not result in a syntactically correct JSON text, then null will be returned instead.

https://stleary.github.io/JSON-java/org/json/JSONObject.html#toString-int-

Reply:
Whoops, sorry, I was so happy it finally worked that I decided to fix all lint warnings, some of them too mechanically…